core/unix/src/TUnixSystem.cxx

```diff
@@ -2857,8 +2857,8 @@ const char *TUnixSystem::GetLinkedLibraries()
       char* longerexe = new char[lenexe + 5];
       strlcpy(longerexe, exe,lenexe+5);
       strlcat(longerexe, ".exe",lenexe+5);
-      delete [] exe;
-      exe = longerexe;
+      // delete [] exe; // dead code, but operation is not allowed
+      exe = longerexe;  // memory leak
```

Comment:
If we are not going to fix the memory leak and the platform is not currently supported, I would put an assert .... or even a static_assert.

Reply:
I guess, memory leak is better than blocking people to try something.
Maybe there are enthusiast who want to re-activate cyginw/gcc combination.

Reply:
@Axel-Naumann opinion?

Reply:
Not a strong one, but applying this and adding an `#error "unsupported platform"` sounds pretty reasonable to me.